src/sqlite_db.cpp

```diff
@@ -93,10 +93,8 @@ class SQLite::impl
             const auto& it_p = miopen_data().find(filepath.filename().string() + ".o");
             if(it_p == miopen_data().end())
             {
-                // Future: Try to load it from the disk system
-                // Also make a config point if Disk I/O is disabled
-                MIOPEN_THROW(miopenStatusInternalError,
-                             "Unknown database: " + filepath.string() + " in internal file cache");
+                MIOPEN_LOG_I("Unknown database: " + filepath.string() + " in internal file cache");
+                return SQLITE_ERROR;
```

Comment:
[Q] Maybe it is worth keeping the old code under `#if !MIOPEN_EMBED_DB`?
```suggestion
#if !MIOPEN_EMBED_DB
                // Future: Try to load it from the disk system
                // Also make a config point if Disk I/O is disabled
                MIOPEN_THROW(miopenStatusInternalError,
                                             "Unknown database: " + filepath.string() + " in internal file cache");
#else
                MIOPEN_LOG_I("Unknown database: " + filepath.string() + " in internal file cache");
                return SQLITE_ERROR;
#endif
```

Reply:
This function is only included when `MIOPEN_EMBED_DB` is `true` so the `!MIOPEN_EMBED_DB` branch will never be compiled.

Reply:
Thanks.